setup.py

```diff
@@ -53,7 +53,7 @@ def find_package_data(where='.', package='', exclude=standard_exclude, exclude_d
       url='https://docassemble.org',
       packages=find_packages(),
       namespace_packages=['docassemble'],
-      install_requires=['docx2python'],
+      install_requires=['docx2python>=1.27.1'],
```

Comment:
Note that Docassemble automatically sets this version requirement--it's not something author controls.